test/service/MarriageAllowanceServiceTest.scala

```diff
@@ -46,17 +46,6 @@ import scala.concurrent.{ExecutionContext, Future}
 //TODO remove the need for TestUtility
 class MarriageAllowanceServiceTest extends UnitSpec with TestUtility with MockitoSugar with GuiceOneAppPerTest {
```

Comment:
Can probably remove GuiceOneAppPerTest

Reply:
tried removing it suites abort with nasty exceptions about class path... will have another try

Reply:
voila